src/oracle/connection/stmt.rs

```diff
@@ -34,10 +34,9 @@ impl Statement {
             let place_holder = limit_clause.split_off(String::from("LIMIT ").len());
             mysql = mysql + &format!("OFFSET 0 ROWS FETCH NEXT {} ROWS ONLY", place_holder);
         }
-        // TODO: this is bad, things will break
-        let is_returning =
-            (sql.starts_with("INSERT") || sql.starts_with("insert")) && sql.contains("RETURNING");
         debug!("SQL Statement {}", mysql);
+        let mut is_select = false;
```

Comment:
Any reason why you define that on the top of the method? Just define it there where the value is assigned?

Reply:
Was not possible due to scope. `is_select` will be determined within `unsafe` and thus it won't get out there

Reply:
That assignment does not belong into that unsafe block :wink: